scripts/install-vm.sh

```diff
@@ -91,7 +91,7 @@ $SUDO mkdir /var/lib/nucliadb
 $SUDO chown nucliadb:nucliadb /var/lib/nucliadb
 
 # Install upgrade script
-cat << EOF | $SUDO tee /usr/bin/upgrade-nucliadb > /dev/null
+cat << 'EOF' | $SUDO tee /usr/bin/upgrade-nucliadb > /dev/null
```

Comment:
The quotes here tell bash to avoid variable substitution in the following script.